docs/getting_started.md

```diff
@@ -14,8 +14,8 @@ work for older releases and other homeservers, but configuration may vary.
 ```sh
 $ git clone https://github.com/matrix-org/matrix-appservice-slack.git
 $ cd Matrix-appservice-slack
-$ npm install
-$ npm run build
+$ yarn install
+$ yarn build
```

Comment:
Only `yarn install` (or just `yarn`) is needed, as `yarn build` gets run automatically by virtue of being the `"prepare"` script in `package.json`.